sdk/eventhub/event-hubs/test/internal/partitionKeyHashMap.spec.ts

```diff
@@ -10,6 +10,7 @@ import { hashPartitionKey } from "../../src/impl/partitionKeyToIdMapper";
  */
 describe("hashPartitionKey", () => {
   it("Generated hashes match the ones the service would have generated", () => {
+    // eslint-disable-next-line @typescript-eslint/no-require-imports
     const expectations: { Key: string; Hash: number }[] = require("./partitionKeyHashMap.json");
```

Comment:
Maybe we should do this while we're visiting this file?

```suggestion
  it("Generated hashes match the ones the service would have generated", async () => {
    // eslint-disable-next-line @typescript-eslint/no-require-imports
    const expectations: { Key: string; Hash: number }[] = await import("./partitionKeyHashMap.json");
```

Reply:
Will follow that up in a subsequent PR